content/v2/domains/templates.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: Templates API | Domains | DNSimple API v2 (Beta)
+excerpt: This page documents the DNSimple templates API v2.
+---
+
+# Templates API
+
+* TOC
+{:toc}
+
+
+## Apply a template {#apply}
+
+    POST /:account/domains/:domain/templates/:template
+
+### Parameters
+
+Name | Type | Description
+-----|------|------------
+`:account` | `integer` | The account id
+`:domain` | `string`, `integer` | The domain name or id
+`:template` | `integer` | The template id
```

Comment:
As far as I remember, the template ID can be either the ID or short name. Am I wrong?